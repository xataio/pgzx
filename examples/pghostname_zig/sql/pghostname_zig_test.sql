CREATE EXTENSION pghostname_zig;

SELECT COALESCE(length(pghostname_zig()), 0) > 0;
