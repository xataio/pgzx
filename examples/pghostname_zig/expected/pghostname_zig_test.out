CREATE EXTENSION pghostname_zig;
SELECT COALESCE(length(pghostname_zig()), 0) > 0;
 ?column? 
----------
 t
(1 row)

